fit/core-reference/pom.xml

```diff
@@ -338,7 +338,7 @@ under the License.
             <deployable>
               <location>${project.build.directory}/${project.build.finalName}</location>
               <pingURL>http://localhost:${cargo.servlet.port}/syncope/cacheStats.jsp</pingURL>
-              <pingTimeout>60000</pingTimeout>
+              <pingTimeout>180000</pingTimeout>
```

Comment:
Why is such change needed?

Reply:
I can pull this back if necessary but this is a "backport" from the master where it allows folks with less-capable machines to run fits with more forgiving timeouts. Depending on availability of system resources (specially I think after one's run fits a few times), the tests seems to start timing out under the old timeout value before Syncope has had a chance to bring itself up.

Reply:
Ok; after all it does not provoke any bad consequence.